dataframely/columns/_base.py

```diff
@@ -67,6 +70,10 @@ def __init__(
                 internally sets the alias to the column's name in the parent schema.
             metadata: A dictionary of metadata to attach to the column.
         """
+
+        if nullable and primary_key:
+            warn_no_nullable_primary_keys()
```

Comment:
Shouldn't we already fail hard in this case? I thought we ignore the nullability property if primary key is set.